### ISSUE TITLE
dev experience: checks for crowdin contributions

### ISSUE THREAD

**Issue by bahrmichael** [bahrmichael/factorio-tycoon#268]
We needed a patch in 0.4.3 to fix a translation error I made. There's also the issue that translations could break when someone omits the key from the `key=value` structure.

@winex made a good suggestion:

> a simple regex (grep -E style) would be sufficient like "^[^\s]+=(.*)$" to match r-value, but idk if it works and where to place it...

We could put that into a github actions workflow that runs when a PR from the crowdin branch (`i18n` i think) is create against `main`.

**Comment by winex** [comment]
looks like this regex extracts groups and keys nicely:
```
$ grep -E -o '^(\[^\s]*\]$|[^=]+=)' locale/en/*.cfg
```
but then we need to compare every other lang file against this list probably to some hacky bash one-liner like:
```
$ diff -q <(cat_this_list) <(grep_again_for_each_file)  # TODO
```
diff has some very useful options:
```
       -q, --brief
              report only when files differ
       -s, --report-identical-files
              report when two files are the same
```

i just don't get all the logic, yet... :(

**Comment by winex** [comment]
@bahrmichael this can be closed as `crowdin-check` was merged (#272 #281) and is working fine on github!

maybe reopen it again if we get any related issue...
